README.md

```diff
@@ -19,6 +19,10 @@ hibernate.connection.password=ENC(9ffpQXJi/EPih9o+Xshm5g==)
 
 Specify the encryption key using the `UP_JASYPT_KEY` environment variable.
 
+### [Modal Notification](docs/modal.md)
+Modal Notification is a single notice that will be presented until the user clicks the `Accept` button. This feature could be used for a Terms of Service pop-up that must be accepted before
+accessing the portal.
```

Comment:
It's great to get info about the model display strategy into the documentation, but much of this sentence has changed...
  - Can show a single notice or a series of notices
  - The button(s) don't necessarily say 'ACCEPT'